### DISCUSSION TITLE
Library re-indexing

### DISCUSSION THREAD

**Discussion by easiwriter** [pointfreeco/swift-composable-architecture#3385] [Q&A]
I just refactored to change a name. I then did the same for another name and Xcode set off to index/prepare all the TCA libraries before displaying the refactor results. This indexing happens whenever I display previews or run a test or compile. Is there some way to stop this when it isn't needed?

**Comment by mbrandonw** [comment]
Hi @easiwriter, I don't think this has anything to do with our library. This is just general Xcode behavior. You might have better luck getting help from Apple's developer forums.
